recipes.adoc

```diff
@@ -168,6 +168,14 @@ endif::[]
 [id='sec.beverages']
 == Beverages
 
+ifdef::env-github[]
+<<recipes/beverages/chia_fresca.adoc#, Chia Fresca>>
+endif::[]
+ifndef::env-github[]
+include::recipes/beverages/chia_fresca.adoc[]
```

Comment:
the filename is chia-fresca, but you've referenced the section id of chia_fresca here (and 3 lines above), this will need to be fixed